### ISSUE TITLE
Fatal Exception: java.lang.SecurityException: One of RECEIVER_EXPORTED or RECEIVER_NOT_EXPORTED should be specified when a receiver isn't being registered exclusively for system broadcasts

### ISSUE THREAD

**Issue by ay1man4** [qonversion/android-sdk#500]
Hello,

App is crashed with **Android 14** only. Below is error details:

```
Fatal Exception: java.lang.SecurityException: One of RECEIVER_EXPORTED or RECEIVER_NOT_EXPORTED should be specified when a receiver isn't being registered exclusively for system broadcasts
       at android.os.Parcel.createExceptionOrNull(Parcel.java:3057)
       at android.os.Parcel.createException(Parcel.java:3041)
       at android.os.Parcel.readException(Parcel.java:3024)
       at android.os.Parcel.readException(Parcel.java:2966)
       at android.app.IActivityManager$Stub$Proxy.registerReceiverWithFeature(IActivityManager.java:5684)
       at android.app.ContextImpl.registerReceiverInternal(ContextImpl.java:1852)
       at android.app.ContextImpl.registerReceiver(ContextImpl.java:1792)
       at android.app.ContextImpl.registerReceiver(ContextImpl.java:1780)
       at android.content.ContextWrapper.registerReceiver(ContextWrapper.java:755)
       at com.android.billingclient.api.zzg.zzc(zzg.java:1)
       at com.android.billingclient.api.zzh.zze(com.android.billingclient:billing@@6.0.0:3)
       at com.android.billingclient.api.BillingClientImpl.startConnection(com.android.billingclient:billing@@6.0.0:48)
       at com.qonversion.android.sdk.internal.billing.QonversionBillingService.startConnection$lambda-9(QonversionBillingService.kt:494)
       at android.os.Handler.handleCallback(Handler.java:958)
       at android.os.Handler.dispatchMessage(Handler.java:99)
       at android.os.Looper.loopOnce(Looper.java:205)
       at android.os.Looper.loop(Looper.java:294)
       at android.app.ActivityThread.main(ActivityThread.java:8176)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:552)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:971)
```

```
Caused by android.os.RemoteException: Remote stack trace:
	at com.android.server.am.ActivityManagerService.registerReceiverWithFeature(ActivityManagerService.java:13908)
	at android.app.IActivityManager$Stub.onTransact(IActivityManager.java:2570)
	at com.android.server.am.ActivityManagerService.onTransact(ActivityManagerService.java:2720)
	at android.os.Binder.execTransactInternal(Binder.java:1339)
	at android.os.Binder.execTransact(Binder.java:1275)

```

**Comment by SpertsyanKM** [comment]
Could you please clarify the steps to reproduce this crash?

**Comment by SpertsyanKM** [comment]
There was an issue with Google Play Billing Library 6.0.0, used in our SDK 5.0.1, so we've released a new version [5.0.2](https://github.com/qonversion/android-sdk/releases/tag/5.0.2) with the upgrade of Billing Library to 6.0.1 which should solve that crash.

Please, give it a try, and feel free to reopen this issue if the problem persists.

Thank you for your report!
